.github/workflows/integration.yml

```diff
@@ -25,99 +64,61 @@ jobs:
         - init-pod-mutator
         - pod-mutator
         - peering-request-webhook-init
-        - crdReplicator
+        - crdreplicator
```

Comment:
can we use the same naming convention? I think that kebab-case fits better than flat-case, just an opinion, though.